Backend/index.js

```diff
@@ -2,58 +2,80 @@ const { PrismaClient } = require("@prisma/client");
 const prisma = new PrismaClient();
 const cors = require("cors");
 const express = require("express");
-const bcrypt = require('bcrypt');
+const bcrypt = require("bcrypt");
 const saltRounds = 14;
 
 const app = express();
 app.use(express.json());
 app.use(cors());
 
-const PORT = process.env.PORT || 5173;
+const PORT = process.env.PORT || 5174;
 
 app.listen(PORT, () => {
   console.log(`Server listening on http://localhost:${PORT}`);
 });
 
+// app.post("/signup", async (req, res) => {
```

Comment:
Can you please clean up the unused code? If you explicitly want to keep them, you can add a comment to briefly explain~